lib/assigner.rb

```diff
@@ -506,6 +508,7 @@ def add_small_action_post(action_code, assign_to, text)
       @assigned_by,
       text,
       bump: false,
+      auto_track: false,
```

Comment:
We need this to prevent the "small action" / "whisper" post that is automatically created when assigning a topic from changing the notification level because the user "replied".